Header header
float32 data0
string unit0
string measurement_type0
float32 data1
string unit1
string measurement_type1